src/swoop/api/models/workflows.py

```diff
@@ -23,6 +25,13 @@ class BaseWorkflow(BaseModel, ABC):
     cache_key_hash_includes: list[StrictStr] = []
     cache_key_hash_excludes: list[StrictStr] = []
 
+    def hash_payload(self, payload):
+        json_filter = JSONFilter(
+            self.cache_key_hash_includes, self.cache_key_hash_excludes
+        )
+        result = json_filter(payload)
+        return hash_dict(result)
+
```

Comment:
The problem with this is that `json_filter` is not created when the workflow model is initialized. The `JSONFilter` constructor validates the includes/excludes lists, so this means an exception for a workflow config issue may not be raised until this method is called. We very specifically want to fail fast on config issues, otherwise that can hide a config problem from the operator and give them a false sense of things being okay ("the application started and is servicing requests, so my config must be good").

The other problem with this solution: instantiating `JSONFilter` is actually rather expensive, and requires paying that cost for _every_ request, of which there could be many millions.

We really need to find a solution here that instantiates the `JSONFilter` as part of the workflow object instantiation to both provide the possibility to have a fast failure and to support instance reuse.

Reply:
I tried this and it seemed to work (per the tests passing):

```
from pydantic import PrivateAttr

class BaseWorkflow(BaseModel, ABC):
    name: StrictStr
    description: StrictStr
    version: StrictInt
    cache_key_hash_includes: list[StrictStr] = []
    cache_key_hash_excludes: list[StrictStr] = []
    _json_filter: JSONFilter = PrivateAttr()

    def __init__(self, **kwargs) -> None:
        super().__init__(**kwargs)
        self._json_filter: JSONFilter  = JSONFilter(
            self.cache_key_hash_includes,
            self.cache_key_hash_excludes,
        )

    def hash_payload(self, payload) -> bytes:
        return hash_dict(self._json_filter(payload))
```